dokka-integration-tests/gradle/build.gradle.kts

```diff
@@ -101,6 +102,13 @@ testing {
 
             targets.configureEach {
                 testTask.configure {
+
+                    inputs.file(templateSettingsGradleKts)
+                    systemProperty(
```

Comment:
If I understand correctly, we set a system property, which is used in `AbstractGradleIntegrationTest`. However, it is not clear to me where we set the second `templateProjectDir` system property.

Reply:
Good point, it's not clear.

I've moved `inputs.file(templateSettingsGradleKts)` to the `registerTestProjectSuite()` util function, so it's next to the `templateProjectDir` input.

This also triggered me to add property names and path sensitivity.

Thanks!